translations.yml

```diff
@@ -192,6 +192,12 @@ parts:
       title: "Label for the scoped knowledge base search setting"
       screenshot: "https://drive.google.com/open?id=1uZaNe12E-A_snTd_7AjcqirB4ROLF4bB"
       value: "Scoped search in Knowledge Base"
+      obsolete: "2025-06-12"
+  - translation:
+      key: "txt.help_center_copenhagen_theme.scoped_knowledge_base_search_label_v2"
+      title: "Label for the scoped knowledge base search setting"
+      screenshot: "https://drive.google.com/file/d/13n_GJuyQJJ7uHAmLD8fobxnmfs3Y9G04/view?usp=drive_link"
+      value: "Scoped search in help center"
```

Comment:
did you run ``` yarn i18n:update-translations ``` ?

We're currently having a translation bug for the errors with non standard locales and I'm wondering if these could stem from us not committing the generated files 

![image](https://github.com/user-attachments/assets/53da84e6-7e87-4005-9df1-a2cdc88f8c93)

Reply:
This has nothing to do with this PR, this PR is adding a source translation string in the YML file for a theme setting.

The `yarn i18n:update-translations` updates the JSON translation files for the React components. 
That said in another PR we should download the locales for the `sk` locale for the `new-request-form` module, the other files are not needed on master since the other modules are used only in the beta branch for now.